frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@monaco-editor/react": "^4.6.0",
```

Comment:
if you're not using the old one can you help do npm uninstall react-monaco-editor (or just remove it from package json) <3

Reply:
i need the old one as a dependency for this to work <3